app/assets/javascripts/delete_idea.js

```diff
@@ -19,6 +20,9 @@ function deleteRow(urlString, removeRow) {
     url:      urlString,
     dataType: 'json',
     success:  function() {
+      // This sort of nesting can get ugly pretty fast, especially if the 
+      // html changes - you won't always be in control of it. Try and think 
+      // of another way to do this.
```

Comment:
You could probably put a class on the elements you want to remove, and then get them. by using $('.element') or something. I try to avoid any sort of walking up and down the DOM like you've put here, because it will invariably break, and there is no separation of concerns. You have to know how the HTML works to look at this and make sense of it, which is mentally straining.